rocketpy/simulation/flight.py

```diff
@@ -1089,6 +1108,25 @@ def __init_post_process_variables(self):
         self._bearing = Function(0)
         self._latitude = Function(0)
         self._longitude = Function(0)
+        # Initialize state derivatives, force and atmospheric arrays
+        self.ax_list = []
+        self.ay_list = []
+        self.az_list = []
+        self.alpha1_list = []
+        self.alpha2_list = []
+        self.alpha3_list = []
+        self.R1_list = []
+        self.R2_list = []
+        self.R3_list = []
+        self.M1_list = []
+        self.M2_list = []
+        self.M3_list = []
+        self.pressure_list = []
+        self.density_list = []
+        self.dynamic_viscosity_list = []
+        self.speed_of_sound_list = []
+        self.wind_velocity_x_list = []
+        self.wind_velocity_y_list = []
```

Comment:
It's said that all of this exists... But it is even worse that they are not private attributes.

Reply:
Agreed